### DISCUSSION TITLE
Does ouch support permissions like `--xattrs`?

### DISCUSSION THREAD

**Discussion by RyanGreenup** [ouch-org/ouch#625] [Q&A]
It's common to use `tar` to backup up a home directory like so:

```bash
mount LABEL=ROOT -o subvol=@ /mnt

# use bsdar if gnu tar compiled without xattrs or acls
cd /mnt
x=/root-tarball.tar.gz
bsdtar --acls --xattrs --preserve-permissions -cvaf ${x} .
```

Users can later extract this with:

```bash
bsdtar --acls --xattrs  --numeric-owner -xpzf ${x} /mnt
```
Is there any support for those properties in _Ouch_? It would be convenient, atleast for me, if there was a command line flag like `-P` / `--preserve-all-permissions`, then one could more simply:

```bash
# Compress
cd /mnt
ouch compress -P . ${x}

# Decompress
cd /
ouch decompress -P ${x} .
```

**Comment by marcospb19** [comment]
If that's OK, I'll close this as a duplicate of the issue you created.
